src/main/java/org/mule/extension/sftp/api/matcher/FileMatcher.java

```diff
@@ -46,8 +46,8 @@ public abstract class FileMatcher<T extends FileMatcher, A extends FileAttribute
   private static final String SIZE_MUST_BE_GREATER_THAN_ZERO_MESSAGE =
       "Matcher attribute '%s' must be greater than zero but '%d' was received";
   public static final boolean DEFAULT_CASE_SENSITIVE = true;
-  protected final ZonedDateTimeLowerOrEqualAssessment FILE_TIME_UNTIL = new ZonedDateTimeLowerOrEqualAssessment();
-  protected final ZonedDateTimeGreaterOrEqualAssessment FILE_TIME_SINCE = new ZonedDateTimeGreaterOrEqualAssessment();
+  protected final ZonedDateTimeLowerOrEqualAssessment fileTimeUntil = new ZonedDateTimeLowerOrEqualAssessment();
+  protected final ZonedDateTimeGreaterOrEqualAssessment fileTimeSince = new ZonedDateTimeGreaterOrEqualAssessment();
```

Comment:
Can you please check if this properties are upper case in main branch too? I have the feeling this might be part of a cherry pick, so we need to make sure the change is propagated correctly

Reply:
I'll check, thanks

Reply:
Indeed they are